.github/workflows/miniziti.yml

```diff
@@ -0,0 +1,34 @@
+name: Test charts in minikube
+on:
+  - pull_request
```

Comment:
It'd be ideal if the triggers were constrained by file path so this would only run when one of the tested charts has changed: controller, router, console, and httpbin.